src/components/modules/admin/ui/pages/CreateLoanRequests.tsx

```
@@ -0,0 +1,181 @@
+import { Card, CardContent } from "@/components/ui/card";
+import { Clock, DollarSign, User, CheckCircle } from "lucide-react";
+import { Skeleton } from "@/components/ui/skeleton";
+import {
+  getAllLoanOffers,
+  approveLoanOffer,
+} from "@/components/modules/dashboard/offer/server/offer.firebase";
+import LoanRequestDetailDialog from "@/components/modules/dashboard/offer/ui/dialogs/LoanRequestDetailDialog";
+import { toast } from "sonner";
+
+export default function LoanOfferRequests() {
+  const [loanOffers, setLoanOffers] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)